sessions/js-advanced-practice/cards-from-form/js/index.js

```diff
@@ -0,0 +1,46 @@
+console.clear();
+
+const cardsContainer = document.querySelector('[data-js=cards]');
+const form = document.querySelector('[data-js=form]');
+
+let cards = [];
+
+form.addEventListener('submit', event => {
+	event.preventDefault();
+
+	const questionElement = form.elements.question;
+	const answerElement = form.elements.answer;
+	const tagsElement = form.elements.tags;
+
+	const newCard = {
+		question: questionElement.value,
+		answer: answerElement.value,
+		tags: tagsElement.value
+			.split(',')
+			.map(tag => tag.trim())
+			.filter(tag => tag.length),
+	};
```

Comment:
With `new FormData` it would look something like that?
```suggestion
        const formData = new FormData(event.target);
        const newCard = Object.fromEntries(formData);
        
        newCard.tags = newCard.tags		
            .split(',')
			.map(tag => tag.trim())
			.filter(tag => tag.length);
```
Not too sure about the splitting.